src/main/java/com/aspire/student/controller/StudentController.java

```diff
@@ -204,6 +204,7 @@ public void getAllStudents(ResourceRequest request, ResourceResponse response) {
     logger.info("getAllStudents()");
     PrintWriter writer = null;
     List<Student> studentList = null;
+    List<Student> studentList2 = null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "studentList2" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)